rest/model/commit_queue.go

```diff
@@ -88,13 +91,33 @@ func ParseGitHubComment(comment string) GithubCommentCqData {
 		if index == 1 {
 			data = parseFirstLine(line)
 		} else if index == 2 {
-			data.MessageOverride = line
+			data.MessageOverride = wrapString(line, commitMessageLineLength)
 		}
 	}
 
 	return data
 }
 
+// wrapString manually splits a string into smaller chunks and appends them to a new string.
+func wrapString(str string, limit int) string {
+	var b strings.Builder
+	words := strings.Fields(str)
+	currentLine := ""
+	for _, word := range words {
+		if len(currentLine+" "+word) > limit {
+			b.WriteString(currentLine + "\n")
+			currentLine = ""
+		}
+		if currentLine == "" {
+			currentLine += word
+		} else {
+			currentLine += " " + word
```

Comment:
We could use `currentLine = strings.Join(currentLine, word, " ")` to get around the if-else here.